.github/workflows/release-test.yml

```diff
@@ -15,21 +15,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-java@v3
+
+      - name: build & push container image '${{ github.event.inputs.releaseVersion }}'
+        uses: mr-smithers-excellent/docker-build-push@v5
         with:
-          distribution: 'temurin'
-          java-version: '17'
-          java-package: 'jdk'
-      - name: Build
-        run: |
-          git config --global user.email "service@minova.com"
-          git config --global user.name "minova-worker"
-          cd $GITHUB_WORKSPACE/cas.app
-          docker login --username ${{ secrets.DOCKER_USERNAME }} --password ${{ secrets.DOCKER_ACCESS_TOKEN }}
-          echo docker build --tag=${{ secrets.DOCKER_USERNAME }}/aero.minova.cas.app:${{ github.event.inputs.releaseVersion }} --build-arg casVersion=${{ github.event.inputs.releaseVersion }} .
-          docker build --tag=${{ secrets.DOCKER_USERNAME }}/aero.minova.cas.app:${{ github.event.inputs.releaseVersion }} --build-arg casVersion=${{ github.event.inputs.releaseVersion }} .
-          docker push ${{ secrets.DOCKER_USERNAME }}/aero.minova.cas.app:${{ github.event.inputs.releaseVersion }}
-        env:
-          MAIN_GITHUB_RELEASE_USERNAME: ${{ secrets.MAIN_GITHUB_RELEASE_USERNAME }}
-          MAIN_GITHUB_RELEASE_TOKEN: ${{ secrets.MAIN_GITHUB_RELEASE_TOKEN }}
-          GITHUB_ACCESS_TOKEN: ${{ secrets.MAIN_GITHUB_RELEASE_TOKEN }}
+          image: 'aero.minova.cas.app'
+          tags: ${{ github.event.inputs.releaseVersion }}
+          registry: 'docker.io'
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_ACCESS_TOKEN }}
+          buildArgs: 'casVersion=${{ github.event.inputs.releaseVersion }}'
+          directory: $GITHUB_WORKSPACE/cas.app
```

Comment:
Ungetestet, da ich mich traue einfach etwas auf `docker.io` zu schieben. Evtl. zusammen testen?

Reply:
In diesem Repo sind keine Geheimnisse drinnen,
also kann man da ruhigen Gewissens nach zum testen pushen.

Reply:
Ich glaube, dieser Workflow ist veraltet und kann sogar gelöscht werden.
Es fehlen hier sowieso die Build-Befehle.